spec/legacy/unit/journal_spec.rb

```diff
@@ -37,21 +37,19 @@
     end
   end
 
-  it 'should create should send email notification' do
-    ActionMailer::Base.deliveries.clear
+  it 'create should send email notification' do
     issue = WorkPackage.find(:first)
     if issue.journals.empty?
       issue.add_journal(User.current, 'This journal represents the creationa of journal version 1')
       issue.save
     end
-    user = User.find(:first)
-    assert_equal 0, ActionMailer::Base.deliveries.size
```

Comment:
I don't know where this assertion stemmed from.

How can I expect no notifications **after** adding a journal?

I made the test pass, but I changed the expectation quite harshly, so please someone point out if there was a reason for the original assertion, that I don't see...